web/src/pages/Dashboard/Courts/CourtCard.tsx

```diff
@@ -48,20 +48,25 @@ const CourtCard: React.FC<ICourtCard> = ({ id, name }) => {
     watch: true,
   });
 
+  const stake = format(jurorBalance?.[0]);
+  const lockedStake = format(jurorBalance?.[1]);
+
   return (
-    <Card>
-      <StyledBreadcrumb items={[{ text: name, value: 0 }]} />
-      <ValueContainer>
-        <label> Stake: </label>
-        <small>{`${format(jurorBalance?.[0])} PNK`}</small>
-      </ValueContainer>
-      <ValueContainer>
-        <WithHelpTooltip {...{ place: "bottom", tooltipMsg }}>
-          <label> Locked Stake: </label>
-        </WithHelpTooltip>
-        <small>{`${format(jurorBalance?.[1])} PNK`}</small>
-      </ValueContainer>
-    </Card>
+    (stake || lockedStake) !== "0" && (
```

Comment:
here, instead of rendering conditionally, I think it would be better to render a `Skeleton` where the value would go